src/Adapters/FHIRAppointmentAdapter.php

```diff
@@ -286,6 +289,28 @@ public function modelToInterface(FHIRAppointment $fhirAppointment)
                                 $patientId = $x2->getValueString();
                                 $appointmentInterface->setPatientId($patientId->getValue());
                                 break;
+
+                            case "#type":
```

Comment:
@Leo24 If these additional fields are required for an appointment to be properly created and displayed in LibreEHR, can you just set default values for these fields upon creation of the appointment? We don't want the Mobile client to have to provide these additional fields.